inst/app/ui.R

```diff
@@ -0,0 +1,19 @@
+library(shiny)
+
+tagList(
+  navbarPage(
+    title = "MapR",
```

Comment:
you can add the version number here like in:
https://github.com/Pandora-IsoMemo/plotr/blob/d8612fd4a3ca2978db06a572e14afcddc2737ed5/inst/app/ui.R#L6